src/hooks/useEdgeSpacing/index.ts

```diff
@@ -0,0 +1,19 @@
+import {useMemo} from 'react';
+import type {EdgeInsets} from 'react-native-safe-area-context';
+import useStyledSafeAreaInsets from '@hooks/useStyledSafeAreaInsets';
+import defaultEdgeSpacing from './defaultEdgeSpacing';
+
+function useEdgeSpacing(padding: Partial<EdgeInsets> = {}): EdgeInsets {
```

Comment:
Change the ` = {}` to ` = CONST.EMPTY_OBJECT`, if not a new object is created every render and the useMemo gets executed